components/SearchModal.tsx

```diff
@@ -224,6 +228,12 @@ const SearchModal: React.FC<SearchModalProps> = ({ setModal }) => {
 
     React.useEffect(() => {
         const handleNavigation = (e) => {
+            if (e.keyCode === 13 && activeResult.current) {
```

Comment:
convert '>' to ascii explicitly for clarity

Reply:
Updated